### PR TITLE
i/o polish

### PR DESCRIPTION
**Pull request by morethanreal**
fix selected styling, write docs

@cdata PTAL cc @notwaldorf

### CODE REVIEW

File: paper-menu.html [hunk 1]
```diff
@@ -30,41 +65,45 @@
...
       /* FIXME move to paper-styles for next widget */
       opacity: 0.12;
       content: '';
 
-      @apply(--paper-menu-focused-item);
+      @apply(--paper-menu-colored-focused-item-after);
     }
 
-    :host > ::content > *[colored]:focus:after {
+    .content > ::content > *[colored]:focus:after {
```
Comment (cdata) [inline paper-menu.html:96]: Just curious: why did you change this? You mentioned to me before that `:host > ::content` was acceptable.

**Comment by cdata** [comment]
LGTM :+1:

File: paper-menu.html (hunk 1 above)
Comment (morethanreal) [inline paper-menu.html:96]: I had a specificity issue: https://github.com/Polymer/polymer/issues/1598
